collectors/fda_dap/spider.py

```diff
@@ -264,6 +270,38 @@ def _get_document_name(el):
                 if name:
                     return name
 
+        def _fix_multipart_review_documents(item, default_document_name='Review'):
+            '''Fixes DAP pages with documents split in many parts.
+
+            If this is the case of the current page, this method will merge the
+            multipart documents in a single one with the name of the document
+            minus the "Part X", or the "default_document_name".
+            '''
+            partial_documents = [doc for doc in item['documents']
+                                 if re.search(r'Part \d+', doc['name'])]
+            keys = set([re.sub(r'Part \d+', '', doc['name']).strip()
+                        for doc in partial_documents])
+
+            for key in keys:
+                indexes_docs = [(i, doc) for i, doc in enumerate(item['documents'])
+                                if doc['name'].startswith(key) or doc['name'].endswith(key)]
+                indexes = [index_doc[0] for index_doc in indexes_docs]
+                docs = [index_doc[1] for index_doc in indexes_docs]
+
+                if not indexes_docs:
+                    continue  # Log error
+
+                # Flatten all documents' URLs arrays into a single one
+                urls = [url for doc in docs for url in doc['urls']]
```

Comment:
So, this is where at some point in the (near?) future we could add a PDF merger, right? Do we still want to do that?